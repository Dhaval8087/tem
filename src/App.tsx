import React, { Fragment } from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';

import { AllEvents, MyEvents } from './features/events';
import Layout from './shared/Layout';
import { About } from './views/about';

function App() {
  return (
    <BrowserRouter>
      <Fragment>
        {/* <Helmet
      titleTemplate='React TypeScript Boilerplate - %s'
      defaultTitle='React TypeScript Boilerplate'
    /> */}
        <main>
          <Layout children={(
            <Switch>
              <Route path='/allevents' component={AllEvents} />
              <Route path='/myevents' component={MyEvents} />
              <Route path='/about' component={About} />
            </Switch>
          )} />
        </main>
      </Fragment>
    </BrowserRouter>
  );
}

export default App;
