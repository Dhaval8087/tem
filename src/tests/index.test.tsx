import { shallow } from 'enzyme';
import React from 'react';

import App from '../App';

test('renders the component', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
});