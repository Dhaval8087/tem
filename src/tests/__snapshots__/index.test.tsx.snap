// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<BrowserRouter>
  <main>
    <withRouter(Layout)>
      <Switch>
        <Route
          component={
            Object {
              "$$typeof": Symbol(react.memo),
              "WrappedComponent": [Function],
              "compare": null,
              "displayName": "Connect(Component)",
              "type": [Function],
            }
          }
          path="/allevents"
        />
        <Route
          component={
            Object {
              "$$typeof": Symbol(react.memo),
              "WrappedComponent": [Function],
              "compare": null,
              "displayName": "Connect(MyEvents)",
              "type": [Function],
            }
          }
          path="/myevents"
        />
        <Route
          component={[Function]}
          path="/about"
        />
      </Switch>
    </withRouter(Layout)>
  </main>
</BrowserRouter>
`;
